src/main/js/helpers/constants.js

```diff
@@ -168,6 +168,12 @@ export default {
         top: -10,
         bottom: 20
     },
+    DEFAULT_TICKSCOUNT: 7,
+    TICKSCOUNT_MAXLIMIT: 19,
```

Comment:
Any particular reason, why max limit is 19?

Reply:
19 ticks would result in 20 divisions. The example graphs are 7-10 ticks. 20 seemed like a big enough number to accommodate as many ticks the consumer may need.